json-cache/service-worker.js

```diff
@@ -0,0 +1,43 @@
+// [Working example](/serviceworker-cookbook/json-cache/).
+
+var CACHE_NAME = 'dependencies-cache';
+
+self.addEventListener('install', function(event) {
```

Comment:
Although is crystal clear for me, I think the code is undercommented for a recipe. Moreover if we want to add docco generated documentation. I think it deserves some explanations about the rationale behind each decission.